meerk40t/core/elements/element_treeops.py

```diff
@@ -167,24 +167,49 @@ def image_convert_modifier(node, **kwargs):
             type="elem image",
         )
 
+    @tree_operation(_("Remove effect"), node_type=effect_nodes, help="")
+    def remove_effect(node, **kwargs):
+        childs = [e for e in node._children]
+        for e in childs:
```

Comment:
`for e in list(node._children):`

Codacy whines about the unneeded comprehension.